playwright/src/main/java/com/microsoft/playwright/impl/Utils.java

```diff
@@ -237,4 +237,23 @@ static String toJsRegexFlags(Pattern pattern) {
     }
     return regexFlags;
   }
+
+  static void setDriverEnvironmentVariables(ProcessBuilder pb) {
```

Comment:
I'd rather we moved it to Driver.setDriverEnvironmentVariables. `PW_CLI_DISPLAY_VERSION` should also be set in the same method I believe.